tests/python/unittest/test_tir_schedule_compute_inline.py

```diff
@@ -1159,5 +1159,180 @@ def test_reverse_compute_inline_producer_predicate_disallowed():
     )
 
 
+def test_compute_inline_softmax():
```

Comment:
Could we simplify the test cases for better readability?

Reply:
I made some attempts but it didn't turn out smoothly - looks like it's challenging to further minimize the length of the test